.pytool/Plugin/DscCompleteCheck/DscCompleteCheck.py

```diff
@@ -131,3 +128,23 @@ def RunBuildPlugin(self, packagename, Edk2pathObj, pkgconfig, environment, PLM,
         else:
             tc.SetSuccess()
         return overall_status
+
+    @staticmethod
+    def _module_in_dsc(inf: str, dsc: DscParser, Edk2pathObj) -> bool:

```

Comment:
Was the type hint for `Edk2pathObj` left off intentionally?

```suggestion
    def _module_in_dsc(inf: str, dsc: DscParser, Edk2pathObj: Edk2Path) -> bool:

```